contrib/AllSupport.cmake

```diff
@@ -27,6 +29,8 @@ function( target_link_contrib ) # 2 arguments: ARGV0 = project name / ARGV1 = sh
 
 	#liblas support
 	target_link_liblas( ${ARGV0} ${ARGV1} )
+    #PDAL support
```

Comment:
Formatting for whole PR - from CONTRIBUTING: "Indentation is expected to be made in Tabs only, each of size 4."